templates/base/storybook/.storybook/config.js

```diff
@@ -1,32 +0,0 @@
-import React from 'react';
-import { configure, addDecorator, addParameters } from '@storybook/react';
-import { DocsPage, DocsContainer } from '@storybook/addon-docs/blocks';
-import { withKnobs } from '@storybook/addon-knobs';
-import { withA11y } from '@storybook/addon-a11y';
```

Comment:
withKnobs and withA11y should be be available.

Reply:
I have added addons to main.js